.github/workflows/ci.yml

```diff
@@ -395,7 +459,7 @@ jobs:
           flags: cloud
 
   marker-tests:
-    needs: [unit-tests, static-analysis]
+    needs: [unit-tests, static-analysis, check-actor-permissions]
```

Comment:
👍  Some of these will need this, some won't.
But until we separate out the credential-needing marker tests, I think we have to apply it to all of them.